doc/0/controllers/index/create/snippets/create.test.yml

```diff
@@ -0,0 +1,11 @@
+---
+name: index#create
+description: create a new index
+hooks:
+  before: curl -X DELETE kuzzle:7512/nyc-open-data
+  after:
+template: default
+expected: index created
+
+sdk: csharp
```

Comment:
No more need to specify the SDK and version, the test framework is based on the extension of the files to find the right runner :)
Same for all the `.test.yml` files